src/device-registry/routes/api-v2.js

```diff
@@ -3052,18 +3052,16 @@ router.post(
 );
 router.get(
   "/events",
-  oneOf([
-    query("tenant")
-      .exists()
-      .withMessage("tenant should be provided")
-      .bail()
-      .trim()
-      .toLowerCase()
-      .isIn(["kcca", "airqo", "view"])
-      .withMessage("the tenant value is not among the expected ones"),
-  ]),
   oneOf([
     [
+      query("tenant")
+        .optional()
+        .notEmpty()
+        .withMessage("the tenant should not be empty if provided")
+        .trim()
+        .toLowerCase()
+        .isIn(["kcca", "airqo", "view"])
```

Comment:
input validation for the tenant query parameter

Reply:
Thanks @Baalmart 
I notice that we pass tenants as environment variable. Is it possible to use them on line 3063?

Reply:
That is a great suggestion that I will explore as soon as possible. @123MwanjeMike 
Could you shed some light on where we use these tenants as environment variables?
Thanks!